tests/README.md

```diff
@@ -111,11 +114,11 @@ each single case.
 /*
 namespace: Compile
 expectation: Pass
-inputs: 
+inputs:
```

Comment:
This style of input for tests is indeed deprecated, you can still put it in the comment but it is not read. I had forgotten to update the existing compiler tests to move the content into input files, but that is now fixed in #1666.

Reply:
Thanks, I just pushed a commit that removes that section.